docs/index.rst

```diff
@@ -12,7 +12,7 @@ We call Patroni a "template" because it is far from being a one-size-fits-all or
 
 Currently supported PostgreSQL versions: 9.3 to 15.
 
-**Note to Citus users**: Starting from 3.0 Patroni nicely integrates with `Citus <https://www.citusdata.com>`__. Please check :ref:`Citus support <citus>` page for more information.
+**Note to Citus users**: Starting from 3.0 Patroni nicely integrates with the `Citus <https://github.com/citusdata/citus>`__ database extension to Postgres. Please check :ref:`Citus support chapter <citus>` in the Patroni documentation for more info about how to use Patroni high availability together with a Citus distributed cluster.
```

Comment:
1. Missing a "the" immediately before "Citus support"
2. is it OK to call it a "Citus support chapter" given that it's a single page, do you consider the page a "chapter"? I'm not familiar with any terminology best practices you have for the Patroni documentation...